rust/relay/server/src/ebpf.rs

```diff
@@ -1,3 +1,20 @@
+#[derive(clap::ValueEnum, Debug, Clone, Copy)]
+pub enum AttachMode {
```

Comment:
Typically, module declarations are the first thing in a Rust file. Can you move this further down?

Like, below all the `use` statements.

Reply:
👍